realm/realm-annotations-processor/src/test/java/io/realm/processor/RealmNameTest.java

```diff
@@ -99,7 +99,7 @@ public void compileModulesWithConflictingPoliciesForNamedClassesFails() {
     // Check that trying to compile two modules with different policies using a mix of `allClasses`
     // and `classes = { ... }` will fail.
     @Test
-    public void compileModulesWithConflictingPoliciesAndMixedClassDefinitionsFails() {
+    public void compileModulesWithConflictingPoliciesAndRealmAnyClassDefinitionsFails() {
```

Comment:
This rename is wrong. Mixed did not refer to the mixed datatype here.

Reply:
It's amazing you saw these things 🙈